pkg/detectors/datadogtoken/datadogtoken.go

```diff
@@ -95,7 +99,7 @@ func setOrganizationInfo(opt []*options, s1 *detectors.Result) {
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"datadog"}
+	return []string{"datadog", "datadoghq", "ddog-gov"}
```

Comment:
`datadoghq` already contains the word `datadog`, so it isn’t necessary to include it separately. We can just add `ddog` to the list along with `datadog`.